phenopacketlab-restapi/src/main/java/org/monarchinitiative/phenopacketlab/restapi/domain/User.java

```diff
@@ -0,0 +1,42 @@
+package org.monarchinitiative.phenopacketlab.restapi.domain;
+
+import com.google.cloud.spring.data.datastore.core.mapping.Entity;
+import org.springframework.data.annotation.Id;
+import org.springframework.security.authentication.AuthenticationCredentialsNotFoundException;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.util.Assert;
+
+@Entity
+public class User {
+	@Id
+	private Long id;
+
+	private String authId;
+
+	public User(){}
+
+	public User(String authId){
+		Assert.notNull(authId, "Required field authId not found.");
+		this.authId = authId;
+	}
+
+	public User(Authentication authentication){
```

Comment:
I think this constructor should be removed and the logic should be moved into `UserService`.